include/autocheck/reporter.hpp

```diff
@@ -92,8 +92,48 @@ namespace autocheck {
       }
   };
 
+struct gtest_checker
+{
+    std::function<void(std::string const&)> expectfn;
+    std::string msg;
+    ostream_reporter rep;
+    template<class FN>
+    void operator <<(FN const& fn)
+    {
+        fn(rep);
+        expectfn(msg);
```

Comment:
I was going to write out what I think this code should be, but it appears it isn't used? If not, please remove it.